advanced_html_css/responsive_design/media_queries.md

```diff
@@ -64,19 +64,19 @@ You'll often see media queries defined with the `screen` keyword like so:
 }
 ~~~
 
-This is mostly unnecessary, but points toward another very useful capability of media queries: changing styles based on media-type. Everything we've covered so far has been specifically intended for viewing on some kind of screen so specifying `screen` is somewhat redundant. It is possible, however, to create a different set of styles for your website when it is sent to your printer or viewed in print-preview mode by using the `print` keyword.
+This is not necessary but it *does* point toward another very useful capability of media queries: **changing styles based on the media type**. Everything we've covered so far has been specifically intended for viewing on some kind of screen so specifying `screen` is redundant. However, it is possible to create a different set of styles for your website when it is sent to your printer or viewed in print-preview mode by using the `print` keyword.
 
 ~~~css
 @media print {
   /* print styles go here! */
 }
 ~~~
 
-This is not something we're going to focus on in this curriculum, but in some cases it may be something you want to consider taking advantage of. It's fairly common to change some colors (i.e. make things black/white), and add `display: none` to hide elements that are useless in a printed environment (buttons, nav-links, etc).
+This is not something we're going to focus on in this lesson, but it may be something you want to consider taking advantage of in some cases. It's fairly common to change some colors (i.e. make things black/white), and add `display: none` to hide elements that are useless in a printed environment (buttons, nav links, etc).
```

Comment:
We could keep "curriculum"; saying "lesson" might come off as it being covered in another lesson.

Reply:
```suggestion
This is not something we're going to focus on in our curriculum, but it may be something you want to consider taking advantage of in some cases. It's fairly common to change some colors (i.e. make things black/white), and add `display: none` to hide elements that are useless in a printed environment (buttons, nav links, etc).
```
Made the change to your suggestion.